backend/projects/project_registry.yaml

```diff
@@ -183,4 +183,35 @@ Conversation:
             machine_translated_conversation_json: machine_translated_conversation_json
           annotations:
             - conversation_json  
-        
\ No newline at end of file
+    ConversationTranslationEditing:
+      project_mode: "Annotation"
+      label_studio_jsx_file: "conversation/conversation_translation.jsx"
+      input_dataset:
+        class: Conversation
+        fields:
+          - domain
+          - topic
+          - scenario
+          - prompt
+          - speaker_count
+          - speakers_json
+          - machine_translated_conversation_json  
+          - parent_data
+        copy_from_parent: 
+          conversation_json: source_conversation_json          
+      output_dataset:
+        class: Conversation
+        save_type: in_place
+        fields:
+          variable_parameters:
+            - language
+          copy_from_input:
+            domain: domain
+            topic: topic
+            scenario: scenario
+            prompt: prompt
+            speaker_count: speaker_count
+            speakers_json: speakers_json
+            machine_translated_conversation_json: machine_translated_conversation_json
```

Comment:
Do we require these? Since `ConversationTranslationEditing` project_type is imported as well as exported using the same dataset, the assumption is that when creating the dataset itself (by the Celery task as part of translate_conversation job), these fields would have already been copied from the parent. Is it not the case?

Reply:
I think they would already be created but I think @ishvindersethi22 could also comment about this.

Reply:
Done.